framework/auth/oauth_scopes.py

```diff
@@ -110,9 +116,9 @@ class ComposedScopes(object):
     ORGANIZER_WRITE = ORGANIZER_READ + (CoreScopes.ORGANIZER_COLLECTIONS_BASE_WRITE, CoreScopes.NODE_LINKS_WRITE)
 
     # Privileges relating to editing content uploaded under that node # TODO: Add wiki etc when implemented
-    NODE_DATA_READ = (CoreScopes.NODE_FILE_READ, )
+    NODE_DATA_READ = (CoreScopes.NODE_FILE_READ, CoreScopes.ADDONS_READ)
```

Comment:
Should allowing a third party to read project files also inherently mean letting them see addon configuration info?

Even though scopes happen on top of existing permissions controls, it doesn't seem that addon endpoints need to be accessible for everyday file usage.

For the future, it seems like what we really want is a separate scope that handles node configuration management... if that's something we allow third parties to do at all.